mri/operators/proximity/weighted.py

```diff
@@ -78,3 +78,296 @@ def mu(self, w):
         if self.zero_weight_coarse:
```

Comment:
Because we don't want to threshold in the coarse scale. weights are not multipliers of the coefficient, they are threshold. I use the same naming convention than for the `WeightedSparseThreshold` class. (see `mu` setter for reference)